src/overlays/actors/ovl_Bg_F40_Block/z_bg_f40_block.c

```diff
@@ -228,8 +228,8 @@ void BgF40Block_Init(Actor* thisx, PlayState* play) {
     BgF40Block* this = THIS;
 
     Actor_ProcessInitChain(&this->dyna.actor, sInitChain);
-    DynaPolyActor_Init(&this->dyna, DYNA_TRANSFORM_POS);
-    DynaPolyActor_LoadMesh(play, &this->dyna, &object_f40_obj_Colheader_004640);
+    DynaPolyActor_Init(&this->dyna, 1);
+    DynaPolyActor_LoadMesh(play, &this->dyna, &gStoneTowerBlockCol);
```

Comment:
```suggestion
    DynaPolyActor_Init(&this->dyna, DYNA_TRANSFORM_POS);
    DynaPolyActor_LoadMesh(play, &this->dyna, &gStoneTowerBlockCol);
```
Must have been caught in the merge conflict